testnets/epixtestnet/chain.json

```diff
@@ -0,0 +1,109 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "epixtestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://epix.zone/",
+  "pretty_name": "Epix Testnet",
+  "chain_type": "cosmos",
+  "chain_id": "epix_4243-0",
+  "bech32_prefix": "epix",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Great call out. This is now complete: https://github.com/satoshilabs/slips/pull/1799